tools/RazorPageGenerator/Program.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.AspNetCore.Razor;
+using Microsoft.AspNetCore.Razor.CodeGenerators;
+
+namespace RazorPageGenerator
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            if (args == null || args.Length != 1)
+            {
+                Console.WriteLine("Invalid argument(s).");
+                Console.WriteLine("Usage:   dotnet razorpagegenerator <root-namespace-of-views>");
+                Console.WriteLine("Example: dotnet razorpagegenerator Microsoft.AspNetCore.Diagnostics.Views");
```

Comment:
The example here isn't correct anymore, right? Might as well change to the new name?